packages/analytics/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.0.18-beta.1
+
+-   chore(suite): depcheck enabled (2206f19f2e)
```

Comment:
@PeKne please next time be more cautions when writing commit messages. we use them to generate changelogs, and your changes were not scoped to `suite` only. https://github.com/trezor/trezor-suite/commit/2206f19f2ef0a1f2f0ed06e803ba1c7571d95bba

more appropriate would be something like "chore:..." without a scope, or maybe "chore(repo):..." or something like this.